sklearn/cluster/tests/test_bicluster.py

```diff
@@ -165,44 +167,45 @@ def test_bistochastic_normalize():
             assert issparse(scaled)
 
 
-def test_log_normalize():
+def test_log_normalize(global_random_seed):
     # adding any constant to a log-scaled matrix should make it
     # bistochastic
-    generator = np.random.RandomState(0)
+    generator = np.random.RandomState(global_random_seed)
     mat = generator.rand(100, 100)
     scaled = _log_normalize(mat) + 1
     _do_bistochastic_test(scaled)
 
 
-def test_fit_best_piecewise():
-    model = SpectralBiclustering(random_state=0)
+def test_fit_best_piecewise(global_random_seed):
+    model = SpectralBiclustering(random_state=global_random_seed)
     vectors = np.array([[0, 0, 0, 1, 1, 1], [2, 2, 2, 3, 3, 3], [0, 1, 2, 3, 4, 5]])
     best = model._fit_best_piecewise(vectors, n_best=2, n_clusters=2)
     assert_array_equal(best, vectors[:2])
 
 
-def test_project_and_cluster():
-    model = SpectralBiclustering(random_state=0)
+def test_project_and_cluster(global_random_seed):
+    model = SpectralBiclustering(random_state=global_random_seed)
     data = np.array([[1, 1, 1], [1, 1, 1], [3, 6, 3], [3, 6, 3]])
     vectors = np.array([[1, 0], [0, 1], [0, 0]])
     for mat in (data, csr_matrix(data)):
         labels = model._project_and_cluster(mat, vectors, n_clusters=2)
         assert_almost_equal(v_measure_score(labels, [0, 0, 1, 1]), 1.0)
 
 
-def test_perfect_checkerboard():
+def test_perfect_checkerboard(global_random_seed):
     # XXX Previously failed on build bot (not reproducible)
-    model = SpectralBiclustering(3, svd_method="arpack", random_state=0)
+    random_state = global_random_seed
```

Comment:
Oops, actually suggestions got lost in the tab-shuffle.

---
A single comment before merging: `global_random_seed` must in place of `random_state`.